tests/cpp_unit_tests/test_tap_position_optimizer.cpp

```diff
@@ -313,13 +316,15 @@ TEST_CASE("Test Transformer ranking") {
             using vertex_iterator = boost::graph_traits<pgm_tap::TransformerGraph>::vertex_iterator;
 
             // Grid with multiple sources and symetric graph
-            pgm_tap::TrafoGraphEdges const edge_array = {{0, 1}, {1, 2}, {2, 3}, {3, 4}, {4, 5}};
-            pgm_tap::TrafoGraphEdgeProperties const edge_prop{
-                {{0, 1}, 1}, {{1, 2}, 1}, {{2, 3}, 1}, {{3, 4}, 1}, {{4, 5}, 1}};
-            std::vector<pgm_tap::TrafoGraphVertex> vertex_props{{true}, {false}, {false}, {false}, {false}, {true}};
```

Comment:
This test is reworked due to the theoretical limit of rank diff == 1. Theory needs revisiting.